tests/e2e/e2e_test.go

```diff
@@ -142,39 +142,40 @@ var _ = ginkgo.BeforeSuite(func() {
 	utils.SetSkipNetworkRunnerShutdown(skipNetworkRunnerShutdown)
 	utils.SetClient(runnerCli)
 
-	ginkgo.By("calling start API via network runner with skipNetworkRunnerStart "+fmt.Sprint(skipNetworkRunnerStart), func() {
-		if skipNetworkRunnerStart {
-			utils.Outf("{{green}}skipped 'start'{{/}}\n")
-		} else {
-			utils.Outf("{{green}}sending 'start' with binary path:{{/}} %q\n", utils.GetExecPath())
-			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
-			resp, err := runnerCli.Start(
-				ctx,
-				utils.GetExecPath(),
-				runner_sdk.WithPluginDir(utils.GetPluginDir()),
-				runner_sdk.WithGlobalNodeConfig(fmt.Sprintf(`{"log-level":"%s"}`, avalanchegoLogLevel)),
-				runner_sdk.WithNumNodes(5),
-				runner_sdk.WithBlockchainSpecs(
-					[]*runner_sdk_rpcpb.BlockchainSpec{
-						{
-							VmName:  vmName,
-							Genesis: utils.GetVmGenesisPath(),
-						},
-					},
-				))
-			cancel()
-			gomega.Expect(err).Should(gomega.BeNil())
-			utils.Outf("{{green}}successfully started:{{/}} %+v\n", resp.ClusterInfo.NodeNames)
-		}
-	})
+	os.Setenv("AVALANCHEGO_PATH", avalanchegoExecPath)
```

Comment:
why is this necessary? Could we add a comment?

Reply:
Done. We need to set this for the solidity suites.go, as we grab the Avalanchego path there. This was never set anywhere and causes the network to not start up properly for `startSubnet`.